pkg/client/client.go

```diff
@@ -95,6 +95,11 @@ type ConmonServerConfig struct {
 
 	// Tracing can be used to enable OpenTelemetry tracing.
 	Tracing *Tracing
+
+	// ShutdownDelay an be used to automatically stop the conmonrs server after DELAY seconds of inactivity.
```

Comment:
Can we document what _inactivity_ means? Like, which actions are considering conmon-rs to be active?

Reply:
+1 to what inactivity means. 

What use cases is this feature trying to solve?